subtests/docker_cli/login/login.py

```diff
@@ -258,8 +261,6 @@ def postprocess(self):
         cmdresult = self.sub_stuff['cmdresult']
         OutputGood(cmdresult, ignore_error=True)
         mustfail(cmdresult, 1)
-        self.failif_not_in("no successful auth challenge", cmdresult.stderr,
-                           "stderr from failed docker login")
         self.failif_not_in("401 Unauthorized", cmdresult.stderr,
                            "stderr from failed docker login")
```

Comment:
Is that going to break in older versions of docker, since they'll (presumably) spit out the old "no successful" message?